### ISSUE TITLE
损失函数需要再求一个batch平均值

### ISSUE THREAD

**Issue by kodakfu** [luozhouyang/DeepSE#3]
`loss = tf.keras.losses.sparse_categorical_crossentropy(labels, cosine, from_logits=True)`    
https://github.com/luozhouyang/DeepSE/blob/a7c47c5146827d50bc46a8ec30da6ee651a0c6b8/deepse/simcse.py#L109

这里的loss实际上是对所有的损失求和，需要再做一个reduce_mean操作

**Comment by kodakfu** [comment]
> ```
> `loss = tf.keras.losses.sparse_categorical_crossentropy(labels, cosine, from_logits=True)`    
> ```
> 
> https://github.com/luozhouyang/DeepSE/blob/a7c47c5146827d50bc46a8ec30da6ee651a0c6b8/deepse/simcse.py#L109
> 
> 这里的loss实际上是对所有的损失求和，需要再做一个reduce_mean操作

是否求平均的loss估计都可以
